### PR TITLE
Adding Appsmith

### PR DESCRIPTION
**Pull request by mrboywunder**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by mvkaran** [comment]
Thanks for your contribution, @mrboywunder! As per contribution guidelines, please add the item to the end of the list.

**Comment by mvkaran** [comment]
@mohanarpit - Please confirm whether Appsmith can be added to the Made in India collection :)

File: collections/made-in-india/index.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 items:
+ - appsmithorg/appsmith
```
Comment (mvkaran) [inline collections/made-in-india/index.md:3]: Thanks for your contribution! As per contribution guidelines, please move this to the end of the list.

**Review by mvkaran — DISMISSED** [review]

**Comment by mohanarpit** [comment]
> @mohanarpit - Please confirm whether Appsmith can be added to the Made in India collection :)

Absolutely! Thank you so much for considering us.

File: collections/made-in-india/index.md (hunk 1 above)
Comment (mrboywunder) [reply collections/made-in-india/index.md]: Resolved.

**Review by kenyonj — CHANGES_REQUESTED** [review]
Thank you for your contribution! Currently we are only accepting additions to collection lists with the new item appended to the end of the list. Please make this change and we can merge it in 😄 .

File: collections/made-in-india/index.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ items:
  - kadalu/kadalu
  - krishdevdb/reseter.css
  - project-sunbird
+ - appsmithorg/appsmith
  - pupilfirst/pupilfirst
```
Comment (kenyonj) [inline collections/made-in-india/index.md:27]:
```suggestion
 - pupilfirst/pupilfirst
 - appsmithorg/appsmith
```

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Pull request merged**